charts/polkadot-watcher/templates/alertrules.yaml

```diff
@@ -44,12 +44,20 @@ spec:
       labels:
         severity: warning
         origin: {{ .Values.origin }}
-    - alert: ValidatorOfflineSession
+    - alert: ValidatorOfflineSessionLong
       annotations:
         message: 'Target {{`{{ $labels.name }}`}} has either not authored any block or sent any heartbeat yet in this session'
       expr: polkadot_offline_validator_session_reports_state > 0
```

Comment:
Before the setup was 3 minutes... I realized that we had a lot of criticals solving themselves after about five minutes. I thought to place a Short warning at 3+5 and a critical at 10

Reply:
Ok sounds good